### ISSUE TITLE
NPE while trying to get connectionId

### ISSUE THREAD

**Issue by vovkab** [facebook/stetho#135]
I'm trying to use stheto with glide, but images is not loading. 
I starting to look into source code and looks like there is npe while trying to get connection id:

**StethoInterceptor.java:49**
```java
response = chain.proceed(request);  // now connection is closed
```

**StethoInterceptor.java:62**
```java
Connection connection = chain.connection(); // so this is null now
```

**NetworkEventReporterImpl.java:137**
```java
// and this is triggers npe in connectionId()
responseJSON.connectionId = response.connectionId(); 
```

**Comment by jasta** [comment]
That seems reasonable, but strange we aren't seeing this more commonly.  Seems easy enough to fix however by just sampling the connection id before proceed and then recalling it.

**Comment by vovkab** [comment]
@jasta thanks

I looked again, maybe it is not really a bug. But maybe something that could be nice to fix anyway.
So I've been trying stetho for some time, and it didn't work for me with glide, so I start trying to use `interceptors()` and `networkInterceptors()` and today I decided to try it again and updated all libs: stetho, glide-okhttp, okhttp, and looks like I left `interceptors()`.

Changing it to `networkInterceptors()` helped, now it works!
